ledger/catchupaccessor.go

```diff
@@ -166,6 +167,14 @@ type catchpointCatchupAccessorImpl struct {
 	nextExpectedAccount basics.Address
 }
 
+// catchpointAccountResourceCounter keeps track of the resources processed for the current account
```

Comment:
there is a copy in store/encodedAccountsIter.go

Reply:
Yes, it's an accumulator used during iteration.
It seemed strange to export the type from store since it makes more sense for it to live here.